python/S3_VPC_ENDPOINT_ENABLED/S3_VPC_ENDPOINT_ENABLED_test.py

```diff
@@ -0,0 +1,335 @@
+        EC2_CLIENT_MOCK.describe_vpcs = MagicMock(return_value=describevpc_return)
+        describevpcendpoints_return = {
+            "VpcEndpoints": [
+                {
+                    "VpcEndpointId": "vpce-03ff48d1f2709aa88",
+                    "VpcEndpointType": "Interface",
+                    "VpcId": "vpc-0056487b",
+                    "ServiceName": "com.amazonaws.us-east-1.secretsmanager",
+                    "State": "available",
+                    "PolicyDocument": "{\n  \"Statement\": [\n    {\n      \"Action\": \"*\", \n      \"Effect\": \"Allow\", \n      \"Principal\": \"*\", \n      \"Resource\": \"*\"\n    }\n  ]\n}",
+                    "RouteTableIds": [],
+                    "SubnetIds": [
+                        "subnet-711b7d3b"
+                    ],
+                    "Groups": [
+                        {
+                            "GroupId": "sg-ceee1485",
+                            "GroupName": "default"
+                        }
+                    ],
+                    "PrivateDnsEnabled": 'True',
+                    "NetworkInterfaceIds": [
+                        "eni-0d20d888cf4ecb0ef"
+                    ],
+                    "DnsEntries": [
+                        {
+                            "DnsName": "secretsmanager.us-east-1.amazonaws.com",
+                            "HostedZoneId": "ZI25M5QAN720W"
+                        },
+                        {
+                            "DnsName": "vpce-03ff48d1f2709aa88-oqdrglvo.secretsmanager.us-east-1.vpce.amazonaws.com",
+                            "HostedZoneId": "Z7HUB22UULQXV"
+                        },
+                        {
+                            "DnsName": "vpce-03ff48d1f2709aa88-oqdrglvo-us-east-1b.secretsmanager.us-east-1.vpce.amazonaws.com",
+                            "HostedZoneId": "Z7HUB22UULQXV"
+                        }
+                    ],
+                    "CreationTimestamp": "2018-08-15T14:29:07.000Z"
+                }
+            ]
+        }
+        EC2_CLIENT_MOCK.describe_vpc_endpoints = MagicMock(return_value=describevpcendpoints_return)
+        resp_expected = []
+        response = RULE.lambda_handler(build_lambda_scheduled_event(), {})
+        resp_expected.append(build_expected_response('NON_COMPLIANT', 'vpc-1234567', compliance_resource_type='AWS::EC2::VPC', annotation='There are no Amazon S3 VPC endpoints present in '+ vpc_id+'.'))
+        assert_successful_evaluation(self, response, resp_expected)
+
+    #Unit test for when Amazon VPC has a AWS S3 VPC present but is not in available state -- Gherkin Scenario 3
+    def test_scenario_3(self):
+        EC2_CLIENT_MOCK.reset_mock()
+        resp_expected = []
+        vpc_id = "vpc-1234567"
+        vpc_return = {'Vpcs':[{'VpcId':'vpc-1234567'}]}
+        EC2_CLIENT_MOCK.describe_vpcs = MagicMock(return_value=vpc_return)
+        describevpcendpoints_return = {
+            "VpcEndpoints": [
```

Comment:
Idem

Reply:
@jongogogo sorry I didn't understand this. can you please elaborate

Reply:
Please remove unnecessary data to improve readability

Reply:
done